lib/ansible/cli/galaxy.py

```diff
@@ -525,7 +531,18 @@ def server_config_def(section, key, required):
             ))
 
         cmd_server = context.CLIARGS['api_server']
-        cmd_token = GalaxyToken(token=context.CLIARGS['api_key'])
+        stdin_api_key = context.CLIARGS['stdin_api_key']
+        api_key = context.CLIARGS['api_key']
+        if api_key:
+            display.warning("Using '--token' or '--api-key' via the cli is insecure. "
+                            "Please consider using '--stdin-token' or '--stdin-api-key'")
+        if stdin_api_key:
+            try:
+                api_key = getpass.getpass(prompt="Galaxy Token:")
```

Comment:
this is really 'prompting for a secret', we normally name these actions --ask and have a common method in base CLI class already for these